bindings/pyroot/pythonizations/python/ROOT/_pythonization/_thistpainter.py

```diff
@@ -0,0 +1,67 @@
+
+################################################################################
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+r"""
+/**
+\class THistPainter
+\brief \parblock \endparblock
+\htmlonly
+<details open>
+<summary  style="font-size:20px; color: #425788;"><b>Python interface</b></summary>
+<div class="pyrootbox">
+\endhtmlonly
```

Comment:
We have recently merged changes that introduce a macro which makes this code redundant, use `\pythondoc THistPainter`, example shown at https://github.com/root-project/root/blob/e8aadfc845c3e98ba88e6893eae8832f8fd2296e/bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tfile.py#L13